app/src/DataFetcher/Misc/Logger.php

```diff
@@ -23,11 +23,14 @@ public function log(string $str)
         if (PHP_SAPI === 'cli') {
             // vendor/bin/sake dev/tasks
             echo rtrim($str, "\n") . "\n";
-        } elseif (Controller::has_curr()) {
-            $class = get_class(Controller::curr());
-            if ($class == TaskRunner::class) {
-                // http://website.test/dev/tasks/MyTask
-                echo "$str<br>\n";
+        } else {
+            $controller = Controller::curr();
+            if ($controller) {
+                $class = get_class(Controller::curr());
```

Comment:
Not super tidy fetching the controller twice but it works and is just for us so I won't ask to change it.